sdk/reya_rest_api/client.py

```
@@ -250,6 +251,31 @@ def owner_wallet_address(self) -> str:
+        for price in prices:
+            if price.asset.upper() == target:
+                return price
+
+        available_assets = ", ".join(sorted(price.asset for price in prices))
+        raise RuntimeError(f"Asset oracle price not found for {asset}. Available assets: {available_assets}")
+
+    async def get_market_mark_price(self, symbol: str) -> str:
+        """Fetch a perp market's mark price from /perpMarket/{symbol}/summary."""
+        summary = await self.markets.get_perp_market_summary(symbol)
+        if summary.mark_price is None:
+            raise RuntimeError(f"Market summary for {symbol} did not include markPrice")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)